utils/common/src/test/java/org/apache/brooklyn/util/text/StringEscapesTest.java

```diff
@@ -115,4 +117,80 @@ public void testJavaLists() {
             JavaStringEscapes.unwrapJsonishListIfPossible("\"\",,x,\"\""));
     }
 
+    @Test
+    public void testJavaListString() {
+        Assert.assertEquals(MutableList.of("hello", "world"),
+            JavaStringEscapes.unwrapQuotedJavaStringList("\"hello\", \"world\"", ","));
+        try {
+            JavaStringEscapes.unwrapQuotedJavaStringList("\"hello\", world", ",");
+            Assert.fail("Should have thrown");
+        } catch (Exception e) { /* expected */ }
+        
+        Assert.assertEquals(MutableList.of("hello", "world"),
+            JavaStringEscapes.unwrapJsonishListStringIfPossible("\"hello\", \"world\""));
```

Comment:
Minor: I'd follow the java convention (and the Brooklyn convention) for indents: 8 spaces. The 4 spaces looks too much like a new code block, rather than a continuation of the previous line.